app/workers/ios_notification_sender.rb

```diff
@@ -0,0 +1,13 @@
+require "houston"
+
+class IOSNotificationSender
+  include Sidekiq::Worker
+
+  def perform(device_token, lesson)
+    notification = Houston::Notification.new(device: device_token['token'])
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.